src/core/CMakeLists.txt

```diff
@@ -904,6 +904,6 @@ if(IVW_EMBED_BUILDINFO)
     target_compile_definitions("inviwo-core" PRIVATE "IVW_EMBED_BUILDINFO")
 else()
     ## ensure that the ini file containing the build info is packaged and installed to the bin folder
-    install(FILES ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/$<CONFIG>/inviwo_buildinfo.ini DESTINATION ${IVW_RUNTIME_INSTALL_DIR} COMPONENT modules CONFIGURATIONS Release Debug)
+    install(FILES ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/inviwo_buildinfo.ini DESTINATION ${IVW_RUNTIME_INSTALL_DIR} COMPONENT modules CONFIGURATIONS Release Debug)
```

Comment:
We need the /$<CONFIG>/ there for multi-configuration builds like visual studio and xcode.  
think one can check GENERATOR_IS_MULTI_CONFIG

Reply:
Ah, makes perfect sense! I'll fix that.

Reply:
Can you please take a look at the updated version? Is that what you had in mind?

Reply:
Great that looks good